assertions/confirmation.go

```diff
@@ -38,6 +38,15 @@ func (m *Manager) keepTryingAssertionConfirmation(ctx context.Context, assertion
 		log.Error("Could not get assertion creation info", "err", err)
 		return
 	}
+	if m.enableFastConfirmation {
+		err = m.chain.FastConfirmAssertion(ctx, creationInfo)
+		if err != nil {
+			log.Error("Could not fast confirm latest assertion", "err", err)
```

Comment:
We should continue here, otherwise the success log below will occur

Reply:
Thanks for catching this, done.